package.json

```diff
@@ -14,24 +14,35 @@
     "prebuild": "rm -rf dist cloudformation.yaml",
     "build": "tsc",
     "postbuild": "./bin/package.sh",
-    "upload-artifact": "node-riffraff-artifact"
+    "upload-artifact": "node-riffraff-artifact",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
   },
   "dependencies": {
     "@guardian/src-foundations": "^0.11.0",
+    "@storybook/addon-knobs": "^5.3.8",
```

Comment:
Should these be `devDependencies` I wonder?

Reply:
I think you're right, I've moved all Storybook specific dependencies to `devDependencies`.